tools/flavor-convert/flavortemplates/linux_tpm20_cbnt.template.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
I don't think we should duplicate the flavor-templates (they will get out of date with the intel-secl repo).  Let's just provide the tool/executable and require that the user of the converter to obtain the flavor-templates.  

@ddhawal do you agree?

Reply:
@kwtj43 Do we need to provide the flavor template in user command line instead of keeping it in the tools/flavor-convert/flavortemplates folder?

Reply:
@kwtj43 Agree but as this is just a file, I doubt if we can import it. Variable can be imported. 

@isteffyx Could you please check that up?

Reply:
@ddhawal I have tried to import flavor template files from intel-secl repo. But I am getting "no such file or directory" error while reading.